content/en/docs/concepts/architecture/nodes.md

```diff
@@ -9,11 +9,10 @@ weight: 10
 
 {{% capture overview %}}
 
-A _Node_ is a worker machine in Kubernetes (previously known as a `minion`). A node
-may be a virtual or physical machine, depending on the cluster. Each node contains
-the services necessary to run {{< glossary_tooltip text="Pods" term_id="pod" >}},
-managed by the {{< glossary_tooltip text="control plane" term_id="control-plane" >}}
-components.
+A _Node_ is a worker machine in Kubernetes. A node may be a virtual or physical
+machine, depending on the cluster. Each node contains the services necessary to
```

Comment:
nit: Is the reference to `worker machine` essential?
Instead:
A Node [the object?] represents a virtual or physical machine, ...
A node is a virtual or physical machine ...

Reply:
I've reworded - how's this?